implementation.md

```diff
@@ -419,8 +429,28 @@ is used to discover the properties of the action.
 
 For details on the parameters to an action, see below.
 
+These two methods are a bit different in how they operate. Actions provided by
+a method are limited by the instances, or one globally. For instance, the SFTP
```

Comment:
"Actions provided by a method are limited by the instances, or one globally."
I wish I could provide feedback other than 'this isn't parsing for me' but I can't work out what this means.

Reply:
Unmatched `}` on line 433.

I've tried to rewrite this whole block.